src/components/util.js

```diff
@@ -0,0 +1,92 @@
+const SECOND = 1000;
```

Comment:
This `util.js` is intended for "component utilities". It may have other `util.js` files in other directories.
As long as `components/util.js` is not required by code outside `components/`, this should be fine.